Travis/js/main.js

```diff
@@ -0,0 +1,219 @@
+  { "title" : "Casino Royale", "year" : 2006, "actor" : "Daniel Craig", "gross" : "$669,789,482" },
+  { "title" : "Moonraker", "year" : 1979, "actor" : "Roger Moore", "gross" : "$655,872,400" },
+  { "title" : "Diamonds Are Forever", "year" : 1971, "actor" : "Sean Connery", "gross" : "$648,514,469" },
+  { "title" : "Quantum of Solace", "year" : 2008, "actor" : "Daniel Craig", "gross" : "$622,246,378" },
+  { "title" : "From Russia with Love", "year" : 1963, "actor" : "Sean Connery", "gross" : "$576,277,964" },
+  { "title" : "Die Another Day", "year" : 2002, "actor" : "Pierce Brosnan", "gross" : "$543,639,638" },
+  { "title" : "Goldeneye", "year" : 1995, "actor" : "Pierce Brosnan", "gross" : "$529,548,711" },
+  { "title" : "On Her Majesty's Secret Service", "year" : 1969, "actor" : "George Lazenby", "gross" : "$505,899,782" },
+  { "title" : "The World is Not Enough", "year" : 1999, "actor" : "Pierce Brosnan", "gross" : "$491,617,153" },
+  { "title" : "For Your Eyes Only", "year" : 1981, "actor" : "Roger Moore", "gross" : "$486,468,881" },
+  { "title" : "Tomorrow Never Dies", "year" : 1997, "actor" : "Pierce Brosnan", "gross" : "$478,946,402" },
+  { "title" : "The Man with the Golden Gun", "year" : 1974, "actor" : "Roger Moore", "gross" : "$448,249,281" },
+  { "title" : "Dr. No", "year" : 1962, "actor" : "Sean Connery", "gross" : "$440,759,072" },
+  { "title" : "Octopussy", "year" : 1983, "actor" : "Roger Moore", "gross" : "$426,244,352" },
+  { "title" : "The Living Daylights", "year" : 1987, "actor" : "Timothy Dalton", "gross" : "$381,088,866" },
+  { "title" : "A View to a Kill", "year" : 1985, "actor" : "Roger Moore", "gross" : "$321,172,633" },
+  { "title" : "License to Kill", "year" : 1989, "actor" : "Timothy Dalton", "gross" : "$285,157,191" }
+];
+
+ // does not work
+// let filmTitles = [bondFilms[1].title];
+// 	for (let i = 1; i <= bondFilms.length-1; i++){
+// 	// add to array
+// 	filmTitles.push(bondFilms[i][i])
+// 	}
+
+// console.log(filmTitles)
+
+// Why does this one workk?
+
+let filmTitles = [];
+	for (i in bondFilms){
+	// add to array
+	filmTitles.push(bondFilms[i].title)
+	}
+
+console.log(filmTitles)
+
+// Create a new array oddBonds, of only the Bond films released on odd-numbered years.
+let oddBonds = [];
+	for (i in bondFilms){
+		// check if odd
+		if(bondFilms[i].year % 2==0){
+		// add to array
+		oddBonds.push(bondFilms[i].title)
+	}
+}
+
+console.log(oddBonds);
+
+
+//Determine the total cumulative gross of the Bond franchise, and console.log the result. hint To make the grosses into usable numbers, look into the .replace Javascript method (there are many ways to do this, however). Look into parseInt also.
+
+
+totalGross = 0;
+
+	for (i in bondFilms){
+ 		let cleanGross = bondFilms[i].gross.replace(/\W/g, '');
+		let parsedGross = parseInt(cleanGross, 10);
+		totalGross += parsedGross;
+}
+
+console.log(totalGross.toLocaleString('en-US', { style: 'currency', currency: 'USD' }));
```

Comment:
beautiful -- love that you researched string formatting and made it have pretty output